test_case_metrics.py

```diff
@@ -366,10 +366,10 @@ def adjust_battery_dispatch(test_case):
 
 if __name__ == "__main__":
     # names:
-    available_gen_sources = ['Battery Discharge Power (kW)', 'PV to Grid (kW)', 'Net Wind Generation (kW)']
```

Comment:
I don't think we need to remove this from the list, because it'll give an error if we try to re-run any older test cases with wind. when gen_sources is calculated, it'll remove 'Net Wind Generation (kW)' for any test cases that don't have wind generation

Reply:
Thanks for catching this!